src/operator/quantization/quantized_activation.cc

```diff
@@ -115,6 +115,8 @@ the float32 data into int8.
 .add_argument("max_data", "NDArray-or-Symbol", "Maximum value of data.")
 .add_arguments(ActivationParam::__FIELDS__());
 
+// TODO(zhiyuan): remove below comment once the accuracy issue of this op is addressed.
```

Comment:
The statement is not very clear.  In fact, we need the extra condition to check if we switch on quantized sum and relu on in the network because this kind of OP is not compute-intensive.

Reply:
Yes, it's more clear in your way:)